docs/language/overview.md

```diff
@@ -1699,10 +1675,10 @@ the value `this` refers to the inner sequence generated from the `over` expressi
 This query runs to completion for each inner sequence and emits
 each subquery result as each inner sequence traversal completes.
 
-This structure is powerful because _any_ Zed query can be appear in the body of
+This structure is powerful because _any_ Zed query can appear in the body of
 the lateral scope.  In contrast to the `yield` example, a sort could be
-applied to each sub-sequence in the subquery, where sort
-reads all of values of the subsequence, sorts them, emits them, then
+applied to each subsequence in the subquery, where sort
+reads all of the values of the subsequence, sorts them, emits them, then
```

Comment:
This is a long sentence, so removing a couple words that don't affect its meaning seem like a win.
```suggestion
reads all values of the subsequence, sorts them, emits them, then
```